jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why are these options defined here and in the `.babelrc`, do we need both?

Reply:
Whoops, this one only enables path autocomplete in vscode